jena-arq/src/main/java/org/apache/jena/riot/lang/JsonLDReader.java

```diff
@@ -112,8 +130,21 @@ public void read(InputStream in, String baseURI, ContentType ct, StreamRDF outpu
             IO.exception(e) ;
         }
     }
-    
-    private void read$(Object jsonObject, String baseURI, ContentType ct, final StreamRDF output, Context context) {
+
+    private void readWithJsonLDCtxOptions(Object jsonObject, String baseURI, final StreamRDF output, Context context)  throws JsonParseException, IOException {
+        JsonLdOptions options = getJsonLdOptions(baseURI, context) ;
+        Object jsonldCtx = getJsonLdContext(context);
+        if (jsonldCtx != null) {
+            if (jsonObject instanceof Map) {
+                ((Map) jsonObject).put("@context", jsonldCtx);
```

Comment:
I'm getting warnings on lines 114, 139: these should be suppressed (it makes it easier to see when new warnings appear in the codebase).

```
Map is a raw type. References to generic type Map<K,V> should be parameterized
	JsonLDReader.java	/jena-arq/src/main/java/org/apache/jena/riot/lang	line 139
Type safety: The method put(Object, Object) belongs to the raw type Map. References to generic type Map<K,V> should be parameterized
	JsonLDReader.java	/jena-arq/src/main/java/org/apache/jena/riot/lang	line 139
Unnecessary @SuppressWarnings("rawtypes")
	JsonLDReader.java	/jena-arq/src/main/java/org/apache/jena/riot/lang	line 114
Unnecessary @SuppressWarnings("unchecked")
	JsonLDReader.java	/jena-arq/src/main/java/org/apache/jena/riot/lang	line 114
```

Reply:
I did not change this part of the code. It gives warning [as it gave before](https://github.com/apache/jena/blob/bfce1741cb12f9cf544235d32fba6598bc7341b5/jena-arq/src/main/java/org/apache/jena/riot/lang/JsonLDReader.java#L94).

What do you suggest?